server/notification-providers/whapi.js

```diff
@@ -0,0 +1,46 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class Whapi extends NotificationProvider {
+
+    name = "whapi";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+
+        let okMsg = "Sent Successfully.";
+
+        let apiUrl = notification.whapiApiUrl;
+        let apiToken = notification.whapiAuthToken;
+        let toNumber = notification.whapiToNumber;
+
+        try {
+
+            let config = {
+                headers: {
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                    "Authorization": "Bearer " + apiToken,
+                }
+            };
+
+            let data = {
+                "to": toNumber + "@s.whatsapp.net",
+                "body": msg,
+            };
+
+            let url = apiUrl + "/messages/text";
+
+            await axios.post(url, data, config);
```

Comment:
nit:
- I would prefer to not having the variables extracted as I don't see a good reason for his behaviour
- and using `const` instead of `let`

```suggestion
        try {
            const config = {
                headers: {
                    "Accept": "application/json",
                    "Content-Type": "application/json",
                    "Authorization": "Bearer " + notification.whapiAuthToken,
                }
            };

            const data = {
                "to": notification.whapiToNumber + "@s.whatsapp.net",
                "body": msg,
            };

            const url = notification.whapiApiUrl + "/messages/text";

            await axios.post(url, data, config);
```